CruCentralCoast/Profile/ChooseCampusOrMovement/ChooseCampusCell.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  ChooseCampusCell.swift
+//  CruCentralCoast
+//
+//  Created by Michael Cantrell on 7/22/18.
+//  Copyright © 2018 Landon Gerrits. All rights reserved.
+//
+
+import UIKit
+
+class ChooseCampusCell: UITableViewCell {
+    
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var campusImageView: UIImageView!
+    @IBOutlet weak var movementsLabel: UILabel!
+    
+    private var currentImageLink: String?
+
+    func configure(with campus: Campus, subscribedMovements: [String]) {
+        if campus.movements.count > 1 {
+            self.accessoryType = .disclosureIndicator
+            let filteredMovements = campus.movements.filter({ subscribedMovements.contains($0.id) })
+            self.movementsLabel.text = filteredMovements.map({ $0.name }).joined(separator: ", ")
+        } else {
+            self.accessoryType = subscribedMovements.contains(campus.movements[0].id) ? .checkmark : .none
+            self.movementsLabel.text = ""
+        }
+        self.titleLabel.text = campus.name
+        // TODO: Remove "http:" when backend image links are fixed
+        self.currentImageLink = (campus.imageLink == nil) ? nil : "http:\(campus.imageLink!)"
+        if let imageLink = ((campus.imageLink == nil) ? nil : "http:\(campus.imageLink!)") {
```

Comment:
I haven't looked deeply into the code base, so maybe it's safe here, but force-unwrapping always feels bad to me, I'd rather see some guard with a logger statement if something goes wrong so the app doesn't crash

Reply:
kk I see what's going on here, so once the backend links are fixed you'll change this entire line to be just `if let imageLink = campus.imageLink`, maybe add that into the TODO in case someone else ends up fixing this later that's not you

Reply:
Yeah it is annoying logic that I don't like and want to remove when that is fixed on Firebase.
I will update the comment so others know